src/main/java/uk/gov/hmcts/juror/api/moj/repository/CourtQueriesRepositoryImpl.java

```diff
@@ -47,7 +48,7 @@ public List<CourtLocationDataDto> getCourtDetailsFilteredByPostcode(String first
             .map(tuple -> new CourtLocationDataDto(
                 tuple.get(COURT_CATCHMENT.locCode),
                 WordUtils.capitalizeFully(tuple.get(COURT_LOCATION.name)),
-                null))
+                null, tuple.get(COURT_LOCATION.owner)))
```

Comment:
Why do we have null set here?
Should this be removed from the Dto? or is this used else where

Reply:
it is needed in another place `/all-court-locations`.